config.proto

```diff
@@ -311,36 +312,39 @@ message Config {
     float adc_multiplier_override = 6;
 
     /*
-     * Power management state machine option.
-     * See [power management](/docs/software/other/power) for details.
+     * Wait Bluetooth Seconds\
+     * The number of seconds for to wait before turning of BLE in No Bluetooth states\
```

Comment:
Remove `\`?

Reply:
Got too clever trying to leave markdown line breaks